template.txt

```diff
@@ -0,0 +1,3 @@
+Subject line (try to keep under 50 characters)
```

Comment:
The commit message template is already part of the project. The file is `.gitmessage.txt`.  You don't need to have another commit template file so this one must be removed.